app/services/contact_message_service.rb

```diff
@@ -1,8 +1,10 @@
 class ContactMessageService
+  def self.deliver(email:, message:)
+    data = { from: { type: "user", email: email }, body: message }
+    headers = {'Authorization' => "Bearer #{ENV['INTERCOM_ACCESS_TOKEN']}",
```

Comment:
We should probably remove `app/models/contact_message.rb` as well. As a side note, this means we won't be storing all of our contact messages in our database, which is probably a good thing anyway, right?